tests/functional_tests/test_container_no_root.py

```diff
@@ -3,11 +3,7 @@
 
 from tests.functional_tests.conftest import get_pod_running_containers
 
-container_ignore_list = [
-    "kube-state",
-    "houston",
-    "fluentd",
-]
+container_ignore_list = ["kube-state", "houston", "fluentd", "commander"]
```

Comment:
If you have a trailing comma on the last list item, ruff will format the list as one item per line, otherwise it tries to put it all on one line. Let's leave it as it was and just add commander here so we have a cleaner `git blame`, and because it's easier to read.
```suggestion
container_ignore_list = [
    "commander",
    "kube-state",
    "houston",
    "fluentd",
 ]
```